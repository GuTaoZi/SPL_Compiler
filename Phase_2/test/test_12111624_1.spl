struct hihi{
    int a,b,c[10];
    float d;
};

struct hihi e;

int main(){
    e.c[e.a]=e.c[e.b];
    e.c[e.d]=e.a;
}