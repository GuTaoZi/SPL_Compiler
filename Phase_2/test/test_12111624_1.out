Error type 12 at Line 10: index: 'e.d' should be an integer
